lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -33,7 +33,7 @@
 </script>
 
 <div class="batch-enrollment membership-section">
-  <h2> ${_("Batch Enrollment")} </h2>
+  <h3 class="hd hd-2">${_("Batch Enrollment")}</h3>
```

Comment:
By the way, when you say your goal is to focus on functionality and markup, that does not mean that you ignore the styling implications, and when introducing text such as a header, it's important to understand the impact.